x/staking/types/msg.go

```
@@ -88,17 +81,10 @@ func (msg MsgCreateValidator) GetSignBytes() []byte {
 // ValidateBasic implements the sdk.Msg interface.
 func (msg MsgCreateValidator) ValidateBasic() error {
```

Given that the field is deprecated, we may want to return an error when it is being used?

---

Maybe we can ignore the value without giving error?